nullaway/src/main/java/com/uber/nullaway/Config.java

```diff
@@ -107,6 +107,9 @@
    */
   boolean acknowledgeRestrictiveAnnotations();
 
+  /** @return true if Optional Emptiness Handler is to be used. */
```

Comment:
Can we be more specific on what this does?  We should say this enables checking for `Optional.get()` calls without checking `Optional.isPresent()`

Reply:
Agreed. Either more detail here or maybe even a paragraph or two in the docs and link from here (maybe both?)

Reply:
Done 👍
Will add few lines in the wiki